conda/recipes/cudf/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
   run:
     - python
     - pandas >=0.23.4
-    - numba >=0.41
+    - numba >=0.44
```

Comment:
This version of numba is not yet available in conda-forge, you'll have to add the `numba` channel as well

Reply:
Also please update the `environment` files, too. :)

Reply:
I'm going to tweak the Python to not require 0.44 for the release, and we can consider requiring once Numba 0.44 is released to conda-forge and defaults.